.github/workflows/update-changelog.yaml

```diff
@@ -0,0 +1,38 @@
+name: "Update changelog"
```

Comment:
Was this accidently pooled?

Reply:
I think that was added by backport, gonna remove it